terms.html

```diff
@@ -105,19 +108,17 @@
   </dd>
 
 
-
-  <dt><dfn data-lt="">DID Method</dfn></dt>
+  <dt><dfn data-lt="">DID method</dfn></dt>
 
   <dd>
-A definition of how a specific DID scheme can be implemented
-on a specific distributed ledger or network, including the precise
-method(s) by which DIDs are resolved and deactivated and DID Documents
-are written and updated.
+A definition of how a specific <a>DID scheme</a> can be implemented
+on a specific <a>distributed ledger</a> or network, including the precise
+method(s) by which <a>DIDs</a> are resolved and deactivated and <a>DID
+documents</a> are written and updated. See Section <a href="#did-methods"></a>
```

Comment:
Just a word of warning about doing stuff like this in shared terminology files... &lt;a href="#did-methods">&lt;/a> presumes that the document that this term is pulled into has that section. We may have another spec, like the DID Resolution spec, that pulls in this terminology document and as a result, this code will throw an error (because the DID Resolution spec will most likely not have a #did-methods fragment id, or even worse, it'll point to a section that doesn't talk about what you meant when you created this link.

Reply:
Understood. I was just following the example of other terms that do it the same way. If we think it's a bad idea, then we should create a new PR and change this for all terms.